doc/source/ray-observability/ray-logging.rst

```diff
@@ -77,10 +111,17 @@ This produces the following output:
     (MyActor(index=2) pid=482120) hello there
     (MyActor(index=1) pid=482119) hello there
 
-How to set up loggers
+Customizing worker loggers
 ~~~~~~~~~~~~~~~~~~~~~
-When using ray, all of the tasks and actors are executed remotely in Ray's worker processes. 
-Since Python logger module creates a singleton logger per process, loggers should be configured on per task/actor basis. 
+When using Ray, all tasks and actors are executed remotely in Ray's worker processes.
+
+If you intend to log a message via a worker, use a logger. Get a logger using the filename where the log message is to be emitted; see `the Python logging documentation <https://docs.python.org/3/howto/logging.html#advanced-logging-tutorial>`__ for more information about this.
+
+By default, workers use the standard Ray logging configuration given in `python/ray/log.py` for code that is internal to Ray. Because the Python logger module creates a singleton logger per process, if a worker needs to log a message it should configure logging on a per task or actor basis.
```

Comment:
I don't think this section is necessary (the section above is sufficient). Or just have a single sentence "to change ray library logging config, see X"